hexrd/ui/main_window.py

```diff
@@ -512,6 +514,16 @@ def on_action_export_current_plot_triggered(self):
             HexrdConfig().working_dir = os.path.dirname(selected_file)
             return self.ui.image_tab_widget.export_current_plot(selected_file)
 
+    def on_action_export_to_maud_triggered(self):
+        filters = 'ESG files (*.esg)'
+
+        selected_file, selected_filter = QFileDialog.getSaveFileName(
+            self.ui, 'Export to Maud', HexrdConfig().working_dir, filters)
```

Comment:
```suggestion
        default_path = os.path.join(HexrdConfig().working_dir, "maud.esg")
        selected_file, selected_filter = QFileDialog.getSaveFileName(
            self.ui, 'Export to Maud', default_path, filters)
```

@saransh13 If you want a different default name rather than the last filename used (as mentioned in your other comment), I believe we can modify that here, like shown above.

Reply:
@saransh13 I've updated the code to use this approach now